### ISSUE TITLE
[Binance] BinanceMarketDataServiceRaw.ticker24() consumes not enough weight, causing potential ban

### ISSUE THREAD

**Issue by jwisniewski** [knowm/XChange#4277]
`BinanceMarketDataServiceRaw.ticker24()` consumes not enough permits which can lead to exceed the allowed limit of API calls on Binance. Exact line: https://github.com/knowm/XChange/blob/b45e531138c7f26e5dcafab69adff59dd685ffbd/xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceMarketDataServiceRaw.java#L83

According the current Binance docs, the related API call (api/v3/ticker/24hr) has weight equal 40. Thus, should consume 40 permits of the rate limiter, if I understand everything correctly. The related doc:
https://binance-docs.github.io/apidocs/spot/en/#24hr-ticker-price-change-statistics

TLDR: Looks as a simple change 5 -> 40.

**Comment by cymp** [comment]
Hello, please make a PR.

**Comment by jwisniewski** [comment]
@cymp as requested ;) the PR: #4327

**Comment by jwisniewski** [comment]
The related issue $4327 has been merged. Topic closed.
